pandas-stubs/core/frame.pyi

```diff
@@ -328,6 +333,46 @@ class DataFrame(NDFrame, OpsMixin):
         render_links: _bool = ...,
         encoding: _str | None = ...,
     ) -> _str: ...
+    @overload
+    def to_xml(
+        self,
+        path_or_buffer: FilePath | WriteBuffer[bytes] | WriteBuffer[str],
+        index: bool = ...,
+        root_name: str | None = ...,
+        row_name: str | None = ...,
+        na_rep: str | None = ...,
+        attr_cols: list[HashableT] | None = ...,
+        elem_cols: list[HashableT] | None = ...,
+        namespaces: dict[str | None, str] | None = ...,
+        prefix: str | None = ...,
+        encoding: str = ...,
+        xml_declaration: bool | None = ...,
+        pretty_print: bool | None = ...,
```

Comment:
Docs don't show `None` as option for `pretty_print`

Reply:
Fixed.